python/graphstorm/inference/ep_infer.py

```diff
@@ -45,7 +47,8 @@ def infer(self, loader, save_embed_path, save_prediction_path=None,
             use_mini_batch_infer=False, # pylint: disable=unused-argument
             node_id_mapping_file=None,
             edge_id_mapping_file=None,
-            return_proba=True):
+            return_proba=True,
+            saved_hdf5_embed=False):
```

Comment:
we should use `embedding_format` for future extension.

Reply:
do you suggest to enable embedding_format in this PR or in future update?

Reply:
Use embedding_format instead of saved_hdf5_embed.

Reply:
updated as suggested